components/storage/blockdevice/COMPONENT_QSPIF/QSPIFBlockDevice.cpp

```diff
@@ -1032,7 +1034,7 @@ int QSPIFBlockDevice::_sfdp_detect_best_bus_read_mode(uint8_t *basic_param_table
         }
         is_qpi_mode = false;
         examined_byte = basic_param_table_ptr[QSPIF_BASIC_PARAM_TABLE_FAST_READ_SUPPORT_BYTE];
-        if (examined_byte & 0x40) {
+        if (examined_byte & 0x20) {
```

Comment:
how is this mode fix related to sr_write_size - they are both in the same commit - should they be in separate commits ?

Reply:
Its two small changes, both are related to quad enabling process... though they in different areas. I can separate them if required.

Reply:
Could be fine, what about commit message (issues have details for these fixes to be tter understand them) ?